index.css

```diff
@@ -207,4 +212,126 @@ hr {
   font-size: 20px;
   line-height: 24px;
   padding-top: 1rem;
+  margin-left: 1rem;
+}
+
+.last_section {
+  background-image: url("./images/Contactme.png");
+  min-height: 800px;
+  width: 100%;
+  margin-top: 4rem;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.logo1 {
+  background-image: url("./images/Rectangle4.png");
+  height: 3rem;
+  width: 4rem;
+  margin-left: 19rem;
+}
+
+.logo_container {
+  display: flex;
+  height: 4rem;
+}
+
+.logo2 {
+  background-image: url(./images/Ellipse12.png);
+  height: 21px;
+  width: 34px;
+  margin-left: 8rem;
+}
+
+.logo3 {
+  background-image: url(./images/Ellipse3.png);
+  height: 64px;
+  width: 54px;
+  margin-left: 11rem;
+}
+
+.contact_me {
+  font-size: 2.5rem;
+  text-align: center;
+  padding-right: 1rem;
+  color: white;
+}
+
+.last_para {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: white;
+  left: 24px;
+  width: 334px;
+  margin-left: 14px;
+  margin-bottom: 2rem;
+}
+
+.text1 {
+  height: 3rem;
+  width: 327px;
+  align-items: center;
+  margin-left: 1rem;
+  border-radius: 0.5rem;
+  margin-top: 1rem;
+  border: none;
+  padding: 0.5rem;
+  margin-bottom: 3px;
+}
+
+/* .text2 {
+  height: 2.5rem;
+  width: 19rem;
+  align-items: center;
+  margin-left: 1rem;
+  border-radius: 0.5rem;
+} */
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.